src/Storage/Query/Handler/PagingHandler.php

```diff
@@ -22,20 +22,20 @@ public function __invoke(ContentQueryParser $contentQuery)
 
         foreach ($contentQuery->getContentTypes() as $contenttype) {
             //Find out if we are searching or just doing a simple query
-            if ($searchParam = $contentQuery->getParameter('filter')) {
+            if ($contentQuery->hasParameter('filter')) {
                 $query = clone $cleanSearchQuery;
             } else {
                 $query = $contentQuery->getService('select');
             }
 
             $repo = $contentQuery->getEntityManager()->getRepository($contenttype);
-            $query->setQueryBuilder($repo->createQueryBuilder($contenttype));
+            $query->setQueryBuilder($repo->createQueryBuilder('_' . $contenttype));
```

Comment:
Is this because of #6779 ?
So this is a change that requires Bolt 3.2.15 (upcoming) or higher, right?
Or is it backward compatible?